### PR TITLE
feat: added admitted bed type filter

### LINKED ISSUE #3953: Last admitted to advanced filter on patient tab is not working properly
**Describe the bug**

Duplicate of issue : #3482 

When we use the last admitted to advanced filter on patient tab, "Something went wrong" error popping up.

**To Reproduce**

1. Go to patient tab
2. Click on advance filter
3. Scroll down to last admitted to filter and apply it

**Expected behavior**

- [ ] remove the last admitted to (bed type ) filter entirely
- [ ] replace the filter with a new one called " Admitted to ( Bed Type ) "
- [ ] the new filter "Admitted to (Bed Types)" should be having options : 
- isolation,
- ICU
- Bed with oxygen support
- Regular

**Screenshots**

image

### PR DESCRIPTION
**Pull request by cp-Coder**
# Feature Request

## Proposed Changes

- Replaced `admitted to` with `admitted bed_type` filter

## Associated Issue

- Fixes https://github.com/coronasafe/care_fe/issues/3953
- Fixes https://github.com/coronasafe/care_fe/issues/3482

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

**Comment by aeswibon** [comment]
@nihal467 This PR can bed tested once the backend PR coronasafe/care#1090 merges

**Review by rithviknishad — APPROVED** [review]

File: src/Common/constants.tsx [hunk 1]
```diff
@@ -258,22 +258,13 @@ export const CONSULTATION_SUGGESTION = [
...
-  "ICU with Invasive Ventilator",
-  "Gynaecology Ward",
-  "Paediatric Ward",
-];
-
-export const PATIENT_FILTER_ADMITTED_TO = [
-  { id: "0", text: "Not admitted" },
-  { id: "1", text: "Isolation Room" },
-  { id: "6", text: "Bed with Oxygen Support" },
+  { id: "1", text: "ISOLATION" },
```
Comment (rithviknishad) [inline src/Common/constants.tsx]:
```suggestion
  { id: "1", text: "Isolation" },
```

File: src/Common/constants.tsx [hunk 2]
```diff
@@ -258,22 +258,13 @@ export const CONSULTATION_SUGGESTION = [
...
   { id: "2", text: "ICU" },
+  { id: "3", text: "ICU_WTH_NON_INVASIVE_VENTILATOR" },
+  { id: "4", text: "ICU_WITH_OXYGEN_SUPPORT" },
+  { id: "5", text: "ICU_WITH_INVASIVE_VENTILATOR" },
+  { id: "6", text: "BED_WITH_OXYGEN_SUPPORT" },
+  { id: "7", text: "REGULAR" },
```
Comment (rithviknishad) [inline src/Common/constants.tsx]:
```suggestion
  { id: "2", text: "ICU" },
  { id: "3", text: "ICU with non-invasive ventilator" },
  { id: "4", text: "ICU with oxygen support" },
  { id: "5", text: "ICU with invasive ventilator" },
  { id: "6", text: "Bed with oxygen support" },
  { id: "7", text: "Regular" },
```

**Review by rithviknishad — COMMENTED** [review]
@cp-Coder since the filter works on Integer choices (`id`), the `text` prop can be user friendly ryt?

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@cp-Coder 
image
image

currently, as per staging, we only have four-bed type, which was specifically mentioned in the issue. remove the extra bed type added and mark it for need testing

**Comment by nihal467** [comment]
@cp-Coder

image

the applied filter badges are not showing on the patient page

**Comment by nihal467** [comment]
@cp-Coder can you work on it ASAP

**Comment by nihal467** [comment]
test approved

File: src/Components/Patient/ManagePatients.tsx [hunk 3]
```diff
@@ -355,32 +354,30 @@ export const PatientManager = (props: any) => {
...
                 });
               }}
             ></i>
           </span>
         )
       );
     };
-
-    return qParams.last_consultation_admitted_to_list
+    console.log(qParams.last_consultation_admitted_bed_type_list);
```
Comment (khavinshankar) [inline src/Components/Patient/ManagePatients.tsx]:
```suggestion
```

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**